docs/03_tutorials/how-to-monitor-state.md

```diff
@@ -0,0 +1,81 @@
+
+[[info]]
+|
+This article assumes you have installed the EOSIO software and are familiar with using the EOSIO nodeos and cleos tools. It is recommended that you have completed [the Getting Started section](https://developers.eos.io/eosio-home/docs)
```

Comment:
We need to update some links:
https://developers.eos.io/eosio-home/docs --> https://developers.eos.io/welcome/latest/index